manim/mobject/geometry.py

```diff
@@ -863,6 +864,225 @@ def __init__(self, n=6, **kwargs):
+    with a negligible angle instead (such as 'angle=0.0001').
+
+    There is an alternative version that is instantiated with pre-defined arcs.
+
+    See Also
+    --------
+    :class:`ArcPolygonFromArcs`
+
+    Attributes
+    ----------
+    arcs : :attr:`list`
+        The arcs used to initialize the ArcPolygon.
+
+    Parameters
+    ----------
+    *vertices : Union[:class:`list`, :class:`np.array`]
+        A list of vertices, start and end points for the arc segments.
+    angle : :class:`float`
+        The angle used for constructing the arcs. If no other parameters
+        are set, this angle is used to construct all arcs.
+    radius : Optional[:class:`float`]
+        The circle radius used to construct the arcs. If specified,
+        overrides the specified ``angle``.
+    arc_config : Optional[Union[List[:class:`dict`]], :class:`dict`]
+        When passing a ``dict``, its content will be passed as keyword
+        arguments to :class:`~.ArcBetweenPoints`. Otherwise, a list
+        of dictionaries containing values that are passed as keyword
+        arguments for every individual arc can be passed.
+
+    Examples
+    --------
+    For an example of a Reuleaux triangle see :class:`ArcPolygonFromArcs`
+
+    .. manim:: SeveralArcPolygons
+
+        class SeveralArcPolygons(Scene):
+            def construct(self):
+                a = [0, 0, 0]
+                b = [2, 0, 0]
+                c = [0, 2, 0]
+                ap1 = ArcPolygon(a, b, c, radius=2)
+                ap2 = ArcPolygon(a, b, c, angle=45*DEGREES)
+                ap3 = ArcPolygon(a, b, c, arc_config={'radius': 1.7, 'color': RED})
+                ap4 = ArcPolygon(a, b, c, color=RED, fill_opacity=1,
+                                            arc_config=[{'radius': 1.7, 'color': RED},
+                                            {'angle': 20*DEGREES, 'color': BLUE},
+                                            {'radius': 1}])
+                ap_group = VGroup(ap1, ap2, ap3, ap4).arrange()
+                self.play(*[ShowCreation(ap) for ap in [ap1, ap2, ap3, ap4]])
+                self.wait()
+
+    """
+
+    def __init__(self, *vertices, angle=PI / 4, radius=None, arc_config=None, **kwargs):
+        n = len(vertices)
+        point_pairs = [(vertices[k], vertices[(k + 1) % n]) for k in range(n)]
+
+        if not arc_config:
+            if radius:
+                all_arc_configs = [{"radius": radius} for pair in point_pairs]
+            else:
+                all_arc_configs = [{"angle": angle} for pair in point_pairs]
+        elif isinstance(arc_config, dict):
+            all_arc_configs = [arc_config for pair in point_pairs]
+        else:
+            assert len(arc_config) == n
+            all_arc_configs = arc_config
+
+        arcs = [
+            ArcBetweenPoints(*pair, **conf)
+            for (pair, conf) in zip(point_pairs, all_arc_configs)
+        ]
+
+        VMobject.__init__(self, **kwargs)
```

Comment:
Suggestion (non-blocking):
```suggestion
        super().__init__(**kwargs)
```